app/models/weapon.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+class Weapon < ApplicationRecord
+  include ActiveModel::Dirty
```

Comment:
We don't need this as it is already included in `ApplicationRecord` objects. The module only needs to be included in classes that don't inherit from `ActiveRecord::Base`.